galleries/examples/statistics/histogram_features.py

```diff
@@ -20,14 +20,14 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-np.random.seed(19680801)
+rng = np.random.default_rng(19710717)
```

Comment:
We use a seed of 19680801 throughout the examples as it’s [John Hunters](https://en.m.wikipedia.org/wiki/John_D._Hunter)’s birthday.